pkg/distributor/distributor.go

```diff
@@ -71,7 +71,7 @@ func (cfg *Config) RegisterFlags(fs *flag.FlagSet) {
 
 // RateStore manages the ingestion rate of streams, populated by data fetched from ingesters.
 type RateStore interface {
-	RateFor(streamHash uint64) int64
+	RateFor(tenantID string, streamHash uint64) int64
```

Comment:
I see tenant here and user below, I think we should standardize on tenant as much as possible